include/libtorrent/alert_types.hpp

```diff
@@ -117,10 +117,10 @@ namespace libtorrent
 		virtual std::string message() const override;
 
 		// The peer's IP address and port.
-		tcp::endpoint ip;
+		tcp::endpoint const ip;
```

Comment:
I picked just a handful while doing the cleanup. I didn't want to go full on it before getting some review from you. If you are OK with it, I can go (in another PR) and review all the alerts.